support/vm/testing.go

```diff
@@ -33,6 +33,15 @@ import (
 )
 
 var FIL = big.NewInt(1e18)
+var VerifregRoot address.Address
+
+func init() {
+	var err error
+	VerifregRoot, err = address.NewIDAddress(80)
```

Comment:
Could you please follow up to put this in `singletons.go` too?

Reply:
#1031